masschallenge/django-accelerator
# [AC-5038-fix] -  fix backward compatibility import error

**Test:**

_reproduce:_
- `git checkout AC-5038-mcproject; git pull`.
- on `cfg/base.cfg:107` set rev=AC-5038. 
- install the new revision: `sudo rm -rf src/django-accelerator/;sudo bin/buildout`
- run `bin/django makemigrations mc`, or `bin/django migrate`. See an import error.
  - **This will happen with any migration related command, with any app specified**

_fix_:
- on `cfg/base.cfg:107` set rev=AC-5038-fix. 
- install the new revision: `sudo rm -rf src/django-accelerator/;sudo bin/buildout`
- run `bin/django makemigrations mc`. See "no changes found" as expected.
